packages/nouns-contracts/contracts/NounsAuctionHouseV3.sol

```diff
@@ -238,6 +254,43 @@ contract NounsAuctionHouseV2 is
         emit AuctionMinBidIncrementPercentageUpdated(_minBidIncrementPercentage);
     }
 
+    /**
+     * @notice Set the stream escrow parameters.
+     * @dev Only callable by the owner.
+     */
+    function setStreamEscrowParams(
```

Comment:
Seems nice to reuse the individual setter functions and events, so we enjoy the same requirements and can easily track a single event to see the full history of each of these vars (e.g. not checking ticks > 0 here but we do in the dedicated setter). wdyt?

Reply:
sounds good: b866c04954bc2aad9f586128d9b71a8361e3d852